compose/transfer/twogtp-and-cpp/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Builds an image that can run gogui's twogtp command for connecting two GTP
```

Comment:
Did this file get added to this PR by mistake? I don't see what it has to do with `black`

Reply:
yup, deleted